src/docfx/lib/HtmlSanitizer.cs

```diff
@@ -0,0 +1,200 @@
+                }
+            },
+            { "image", new(StringComparer.OrdinalIgnoreCase) { "alt", "height", "src", "width" } },
+            { "img", new(StringComparer.OrdinalIgnoreCase) { "alt", "height", "src", "width", "align", "hspace", "border", "sizes", "valign" } },
+            { "input", new(StringComparer.OrdinalIgnoreCase) { "type", "value" } },
+            { "ins", new(StringComparer.OrdinalIgnoreCase) { "cite", "datetime" } },
+            { "kbd", null },
+            { "label", new(StringComparer.OrdinalIgnoreCase) { "for" } },
+            { "li", new(StringComparer.OrdinalIgnoreCase) { "value" } },
+            { "mark", null },
+            { "nav", null },
+            { "nobr", null },
+            { "ol", new(StringComparer.OrdinalIgnoreCase) { "reserved", "start", "type" } },
+            { "p", new(StringComparer.OrdinalIgnoreCase) { "align", "dir", "hidden", "lang", "valign" } },
+            { "pre", new(StringComparer.OrdinalIgnoreCase) { "lang" } },
+            { "q", new(StringComparer.OrdinalIgnoreCase) { "cite" } },
+            { "rgn", null },
+            { "s", null },
+            { "samp", null },
+            { "section", null },
+            { "small", null },
+            { "source", new(StringComparer.OrdinalIgnoreCase) { "src", "type" } },
+            { "span", new(StringComparer.OrdinalIgnoreCase) { "dir", "lang" } },
+            { "strike", null },
+            { "strong", null },
+            { "sub", null },
+            { "summary", null },
+            { "sup", null },
+            {
+                "table",
+                new(StringComparer.OrdinalIgnoreCase)
+                {
+                    "align",
+                    "width",
+                    "border",
+                    "valign",
+                    "bgcolor",
+                    "frame",
+                    "cellpadding",
+                    "cellspacing",
+                    "bordercolor",
+                }
+            },
+            { "tbody", new(StringComparer.OrdinalIgnoreCase) { "align", "valign", "width" } },
+            { "td", new(StringComparer.OrdinalIgnoreCase) { "rowspan", "colspan", "align", "width", "valign", "bgcolor", "hidden", "nowrap" } },
+            { "tfoot", null },
+            { "th", new(StringComparer.OrdinalIgnoreCase) { "rowspan", "colspan", "align", "width", "bgcolor", "scope", "valign" } },
+            { "thead", new(StringComparer.OrdinalIgnoreCase) { "align", "valign" } },
+            { "time", new(StringComparer.OrdinalIgnoreCase) { "datetime" } },
+            { "tr", new(StringComparer.OrdinalIgnoreCase) { "align", "valign", "colspan", "height", "bgcolor" } },
+            { "u", null },
+            { "ul", null },
+            { "var", null },
+            { "video", new(StringComparer.OrdinalIgnoreCase) { "src", "width", "height", "preload", "controls", "poster" } },
+            { "wbr", null },
+            {
+                "Global Attributes",
```

Comment:
Use '*' to represent match all tag names, and move this to the top of the list.
```suggestion
                "*",
```

Reply:
done. Will let IA team rename the tag name too.